### PR TITLE
Update people model

### LINKED ISSUE #168: Add new field to people model
On the people model we need a link field, verbose name "Interview Link".

See https://github.com/mozilla/network/issues/359

### PR DESCRIPTION
**Pull request by alanmoo**
Closes #168

### CODE REVIEW

File: app/networkapi/people/models.py [hunk 1]
```diff
@@ -97,6 +97,12 @@ class Person(SortableMixin):
         null=True,
         blank=True,
     )
+    interview_url = models.URLField(
+        max_length=500,
```
Comment (Pomax) [inline app/networkapi/people/models.py]: are you sure about this? it seems a weird limit for a thing that is by spec defined as 4096 characters max

File: app/networkapi/static/people/js/admin.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
-  if(!$("#id_featured").is(':checked')) {
-    $(".field-quote").hide();
-    $(".field-bio").hide();
-  }
-
-  $("#id_featured").click(function() {
-    $(".field-quote").toggle(this.checked);
-    $(".field-bio").toggle(this.checked);
+  var fieldsTiedToFeaturedCards = ".quote, .bio, .interview_url";
+  $(".featured").on('click', function() {
```
Comment (Pomax) [inline app/networkapi/static/people/js/admin.js]: newline between L2 ad L3?

File: app/networkapi/people/models.py (hunk 1 above)
Comment (alanmoo) [inline app/networkapi/people/models.py]: You know, I thought about this as I was typing it but was just looking at the fields above it for reference. Thought maybe it was a weird python/db thing.

Comment (alanmoo) [inline app/networkapi/people/models.py]: Interestingly from the docs: `Like all CharField subclasses, URLField takes the optional max_length argument. If you don’t specify max_length, a default of 200 is used.`

Comment (alanmoo) [inline app/networkapi/people/models.py]: Also, Stackoverflow led me to a a conversation that's somewhat relevant. https://code.djangoproject.com/ticket/19515. So I think I'm going to keep this for now, as I know 500 chars works; If someone needs to post a longer url, they can use a shortener.

Comment (cadecairos) [inline app/networkapi/people/models.py]: typically for other URL fields we've set a max_length of 2048

**Pull request merged**